k8s/upgrades/0.5.x-0.6.0/patch-strategy-recreate.json

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
Can we write yaml instead of json. Advantage being we can write comments about the patch.

Reply:
keeping this as json now, as other templates are in json for now.